Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -235,6 +220,25 @@ namespace Babylon
             return static_cast<bimg::TextureFormat::Enum>(format);
         }
 
+        void FlipY(bimg::ImageContainer* image)
+        {
+            uint8_t* bytes = static_cast<uint8_t*>(image->m_data);
+            uint32_t rowCount = image->m_height;
+            uint32_t rowPitch = image->m_size / image->m_height;
+
+            std::vector<uint8_t> buffer(rowPitch);
+
+            for (size_t row = 0; row < rowCount / 2; row++)
+            {
+                auto frontPtr = bytes + (row * rowPitch);
+                auto backPtr = bytes + ((rowCount - row - 1) * rowPitch);
+
+                std::memcpy(buffer.data(), frontPtr, rowPitch);
```

Comment:
nit: Not really a problem, but I think we could reduce the number of copy operations done here by a third by doing the entire "copy to buffer" operation at once, basically just grabbing the top half of the image and putting that in a buffer in a single operation, then reversing the bottom into the top, then reversing the buffer into the bottom. Created #187 to track this, as it's probably not worth doing in this PR.